OverlayPlugin.Core/EventSources/FFXIVRequiredEventSource.cs

```diff
@@ -242,8 +231,13 @@ struct PartyMember
             // Raw job id.
             public int job;
             public int level;
-            // In immediate party (true), vs in alliance (false).
+            // @deprecated, please use partyType
             public bool inParty;
+            public long contentID;
+            public byte flags;
+            public uint objectID;
+            public ushort territoryType;
+            public string partyType;
```

Comment:
Can you leave a comment as to what values this can have?

Reply:
(sorry about single responses on the remaining comments, for some reason they don't show up in the files changed tab so I can't do a review comment for them).

There's some basic info in the no-party-so-default-to-current-player logic, but the meanings of the exact values aren't known AFAIK.

Reply:
I moved that info to this definition, hopefully that addresses this?

Reply:
Sorry if I am being confused, but where is that in this PR?

Reply:
https://github.com/OverlayPlugin/OverlayPlugin/blob/94a73c0dd80b5d2292395300e1826d6b8d64961a/OverlayPlugin.Core/EventSources/FFXIVRequiredEventSource.cs#L237-L241

Reply:
Ah, sorry.  Maybe github snippet is making this ambiguous?  Thanks for the `flags` comments, but I was talking about the `partyType` string.

Reply:
Yeah, sorry, that was the snippets being weird.

Maybe `partyType` should be an enum that gets `toString`'d on the return event object, to avoid any ambiguity/potential for typos?

That would also serve to centralize possible values.

Reply:
Ooh, that's a great suggestion.

Reply:
This should be more clear now.